docs/decisions/0009-persistence-and-retries-for-events.rst

```diff
@@ -0,0 +1,25 @@
+Persistence and retries for events
+==================================
+
+Status
+------
+
+Pending
+
+Context
+-------
+
+`event-routing-backends` transmits events to configured Learning Record Consumer (LRC) via http protocol in near real-time. A strategy needs to be implemented to handle the case when an LRC's link is down.
+
+Decision
```

Comment:
[Nit] It's good to include a "Consequences" section in ADRs since there's usually always tradeoffs with decisions.

Reply:
Good idea. Added 3 consequences.